BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -75,16 +75,41 @@ public final class BlueprintView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
-    /// Forwarded to the `measure(in:)` implementation of the root element.
+    ///
+    /// Measures the size needed to display the view within then given constraining size,
+    /// by measuring the current `element` of the `BlueprintView`.
+    ///
+    /// If you would like to not constrain the measurement in a given axis,
+    /// pass `0.0` or `.greatestFiniteMagnitude` for that axis, eg:
+    ///
+    /// ```
+    /// // Measures with a width of 100, and no height constraint.
+    /// blueprintView.sizeThatFits(CGSize(width: 100.0, height: 0.0))
+    ///
+    /// // Measures with a height of 100, and no width constraint.
+    /// blueprintView.sizeThatFits(CGSize(width: 0.0, height: 100.0))
+    ///
+    /// // Measures unconstrained in both axes.
+    /// blueprintView.sizeThatFits(.zero)
+    /// ```
+    ///
     override public func sizeThatFits(_ size: CGSize) -> CGSize {
-        guard let element = element else { return .zero }
-        let constraint: SizeConstraint
-        if size == .zero {
-            constraint = SizeConstraint(width: .unconstrained, height: .unconstrained)
-        } else {
-            constraint = SizeConstraint(size)
+        guard let element = element else {
+            return .zero
         }
-        return element.content.measure(in: constraint)
+        
+        return element.content.measure(in: Self.sizeThatFitsMeasurementConstraint(with: size))
+    }
+    
+    private static func sizeThatFitsMeasurementConstraint(with size : CGSize) -> SizeConstraint {
```

Comment:
nit: if this behavior is specific to `sizeThatFits`, can you inline it, or nest this func in `sizeThatFits`?

Reply:
TIL you can nest functions in other functions, whoa

Reply:
Done